eem-sparks/src/main/java/com/nice/eem/sparkathon/AWSS3ClientBuilder.java

```diff
@@ -24,8 +21,8 @@ public static void main(String[] args) {
      */
     public static AmazonS3 buildS3Client() {
         AmazonS3 s3Client;
-        String awsRegion = "us-east-1";
-        BasicAWSCredentials awsCreds = new BasicAWSCredentials(ACCESS_KEY, SECRET_KEY);
+        String awsRegion = null;
+        BasicAWSCredentials awsCreds = new BasicAWSCredentials("XXXXXXXXXXXXXXXXXXXX", "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This looks like an AWS access key.
Don’t embed access keys directly into code. Instead, use the AWS SDK or AWS Command Line Tools to store access keys in a secure location that can be referenced by your code. If an access key is in your code then it could have leaked. Therefore we recommend that you rotate your credentials as soon as possible. Learn more about [Best Practices for Managing AWS Access Keys](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html).